README.md

```diff
@@ -258,17 +275,79 @@ The connector only requires `events:PutEvents` permission as shown in the IAM po
             "Sid": "AllowPutEventsKafkaConnector",
             "Effect": "Allow",
             "Action": "events:PutEvents",
-            "Resource": "arn:aws:events:us-east-1:1234567890:event-bus/kafkabus"
+            "Resource": "<ARN of your event bus>"
         }
     ]
 }
 ```
 
-> **Note**  
+> [!IMPORTANT]
 > If you use the Glue Schema Registry, the IAM role needs additional permissions to retrieve schemas e.g.,
 > using the managed policy `AWSGlueSchemaRegistryReadonlyAccess`. Please refer to the Glue Schema Registry
 > [documentation](https://docs.aws.amazon.com/glue/latest/dg/schema-registry.html).
 
+#### Using different Configuration Profiles per Connector
+
+If you run multiple EventBridge connectors in your Kafka Connect environment, using environment variables or Java system
+properties to configure your connectors means that each connector will be configured with the same IAM permissions. If
+you want to configure multiple connectors with specific (different) IAM profiles from your `config` and `credentials`
+files, the connector configuration option `aws.eventbridge.iam.profile.name` can be used.
+
+With the connector configuration option `aws.eventbridge.iam.profile.name` you specify which profile the specific
+connector will use. 
+
+> [!IMPORTANT]
+> Environment variables, such as `AWS_PROFILE` or AWS access keys always take precedence over the configuration
+> files and **must not** be set for this configuration option to take effect.
+
```

Comment:
Is it possible to check for this using System variables and throw an error when `profile.name` is set?

Reply:
Yes, this is a good idea! Instead of throwing, I'm considering putting a warning instead so that the connector would still work but `profile.name` would be ignored e.g., if `AWS_PROFILE` is set or any of the credential environment variables. WDYT?